x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -224,6 +225,16 @@ export const showView: (state: EndpointState) => 'policy_response' | 'details' =
   }
 );
 
+/**
+ * Returns the Host Status which is connected the fleet agent
+ */
+export const hostStatusInfo: (state: Immutable<EndpointState>) => HostStatus = createSelector(
+  (state) => state.hostStatus,
+  (hostStatus) => {
+    return hostStatus ? hostStatus : HostStatus.ERROR;
```

Comment:
I assume we don't have an `UNKNOWN` type? If we do, would it make more sense here to set it to `UNKOWN`?

Reply:
yup there is no unknown type, do you think it's better to make another type, or just set it to a different type?